Makefile

```diff
@@ -67,10 +68,17 @@ check-format:
 	npx prettier --check "**/*" --ignore-unknown
 
 # ref https://superuser.com/a/1170997
-.PHONY: bundle-python
-bundle-python:
+.PHONY: bundle-python-mac
+bundle-python-mac:
 	ditto -c -k --sequesterRsrc --keepParent ./mockup_package/mockup ./public/mockup.zip
 
+.PHONY: bundle-python-linux
+bundle-python-linux:
+	mkdir -p mockup
+	cp -r mockup_package/mockup/* mockup/
+	zip -r public/mockup.zip mockup
+	rm -rf mockup
+
```

Comment:
Sorry, forgot to say - if we are creating&removing sth in a script, can we follow the convention of putting these into `tmp/` folder? thanks

Reply:
@pkong-ds updated~